### PR TITLE
Fix Tooltip height

### LINKED ISSUE #3266: Fix Tooltip height
### Package name(s)

Core (@salt-ds/core)

### Package version(s)

_No response_

### Description

Tooltips do not match the spec and are too tall.

Tooltip spec: https://www.figma.com/design/0qHqLDEobXhRGb1PzQ0I5q/Tooltip?t=Yhu4PzPwLh4YliZe-0

### Steps to reproduce

_No response_

### Expected behavior

_No response_

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [ ] I am an employee of JPMorgan Chase & Co.

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
closes #3266

### CODE REVIEW

File: packages/core/src/tooltip/Tooltip.css [hunk 1]
```diff
@@ -20,10 +21,14 @@
   font-weight: var(--saltTooltip-fontWeight, var(--salt-text-fontWeight));
   line-height: var(--saltTooltip-lineHeight, var(--salt-text-lineHeight));
   max-width: var(--saltTooltip-maxWidth, 230px);
```
Comment (joshwooding) [inline packages/core/src/tooltip/Tooltip.css:23]: Do you mind checking this value :)

**Comment by origami-z** [comment]
Is this ready?

File: packages/core/src/tooltip/Tooltip.css (hunk 1 above)
Comment (origami-z) [reply packages/core/src/tooltip/Tooltip.css:25]: Captured in #3694

File: .changeset/unlucky-cats-add.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Fixed tooltip not having correct height specificed by design
```
Comment (joshwooding) [inline .changeset/unlucky-cats-add.md:5]:
```suggestion
Fixed `Tooltip` not having correct height.
```

**Review by joshwooding — APPROVED** [review]

**Review by origami-z — APPROVED** [review]

**Pull request merged**